integration-test/src/test/java/org/jboss/pnc/integration_new/BuildConfigurationEndpointTest.java

```diff
@@ -81,4 +84,22 @@ public void shouldPatchBuildConfiguration() throws ClientException, PatchBuilder
         Assert.assertEquals(newDescription2, updated2.getDescription());
     }
 
+    @Test
+    public void shouldCreateBuildConfigRevision() throws ClientException {
+        BuildConfigurationClient client = new BuildConfigurationClient(RestClientConfiguration.getConfiguration(RestClientConfiguration.AuthenticateAs.USER));
+        BuildConfiguration bc = client.getSpecific(100);
+
+        BuildConfiguration newBC1 = bc.toBuilder().description("Updated description.").build();
+        BuildConfiguration newBC2 = bc.toBuilder().buildScript("mvn deploy # Updated script").build();
```

Comment:
I'd rather make both values final local variables.

Reply:
I have no problem with that, but why?

Reply:
IMHO it is better to use final variables in tests rather then copy-paste magic-value

Reply:
Hmm, so you mean it like this
```suggestion
        final BuildConfiguration newBC2 = bc.toBuilder().buildScript("mvn deploy # Updated script").build();
```
Or something else?

Reply:
Oh, I guess you ment to extract the String parametr to final variable.

Reply:
Yes, I mean to extract the String value